main.py

```diff
@@ -71,6 +71,14 @@ def _get_link_from_interface(self, interface):
                 return link
         return None
 
+    @staticmethod
+    def _get_data(req):
+        """Get request data."""
+        data = req.get_json()  # Valid format { "interfaces": [...] }
```

Comment:
data = req.get_json() or {}

Reply:
Thanks for suggestion @gleybersonandrade.